tests/test_page.py

```diff
@@ -122,6 +122,28 @@ def test_page_operations(pdf_path, password):
     page.extract_text()
 
 
+def test_mediabox_expansion_after_rotation():
+    """TODO"""
+
+    pdf_path = RESOURCE_ROOT / "crazyones.pdf"
+    reader = PdfReader(pdf_path)
+    writer = PdfWriter()
+
+    transformation = Transformation().rotate(175)
+    for page_box in reader.pages:
+        page_box.add_transformation(transformation, expand=True)
+        writer.add_page(page_box)
+
+    rotated_pdf_bytes = BytesIO()
+    writer.write_stream(rotated_pdf_bytes)
+
+    expected_pdf_path = RESOURCE_ROOT / "crazyones_rotated_175.pdf"
+    with open(expected_pdf_path, "rb") as f:
+        expected_pdf_bytes = BytesIO(f.read())
+
+    assert rotated_pdf_bytes.getvalue() == expected_pdf_bytes.getvalue()
```

Comment:
I would have just look at the box size as you did in your graphs with a criteria <1E-4

Reply:
That makes sense @pubpub-zz . I've updated the tests.

Due to rounding/interpolation, there's deviation of about 1-ish pixels compared to pillow. I've set an absolute tolerance of 2 pixels instead.

Reply:
Lmk if this looks good. Or if you want me to set the expected dimensions to be what `pypdf` actually ends up with -- and compare with a small tolerance instead.